python/pyspark/sql/functions/builtin.py

```diff
@@ -3696,7 +3696,7 @@ def regr_avgx(y: "ColumnOrName", x: "ColumnOrName") -> Column:
     |           NULL|  NULL|
     +---------------+------+
 
-    Example 3: All paris's y values are null
+    Example 3: All pairs's y values are null
```

Comment:
Is this `pairs's` correct? Shouldn't be `pair's`?

Reply:
Should be `pairs'`

Reply:
ok, let's change it then:
```suggestion
    Example 3: All pairs' y values are null
```